orchestrator-salt/src/main/java/com/sequenceiq/cloudbreak/orchestrator/salt/SaltOrchestrator.java

```diff
@@ -91,6 +91,14 @@
 
     private static final int SLEEP_TIME = 10000;
 
+    private static final int SLEEP_TIME_MS_1000 = 1000;
+
+    private static final int SLEEP_TIME_MS_2000 = 2000;
+
+    private static final int SLEEP_TIME_MS_3000 = 3000;
+
+    private static final int SLEEP_TIME_MS_4000 = 4000;
```

Comment:
our opinion is that the min sleep time should be 5 sec. it's currently 10 so it still would double the poll count

Reply:
Switching to a single 5 second sleep.